README.md

```diff
@@ -1,5 +1,20 @@
 # lexisnexis-api-client
 
+## Tests
+
+**Unit tests**:
+
+```
+make unit_test
```

Comment:
Looks we rethought this. In the makefile looks like they are `make test` and a few lines lower `make test_integration`.

Reply:
Whoops, good catch, thanks